src/backend/utils/load/age_load.c

```diff
@@ -63,57 +63,78 @@ static agtype* create_empty_agtype(void) {
+                                   WAGT_BEGIN_OBJECT, NULL);
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_END_OBJECT, NULL);
 
     return agtype_value_to_agtype(result.res);
 }
 
-agtype* create_agtype_from_list(char **header, char **fields, size_t fields_len, int64 vertex_id) {
+agtype* create_agtype_from_list(char **header, char **fields,
+                                size_t fields_len, int64 vertex_id) {
     agtype_in_state result;
     int i;
 
     memset(&result, 0, sizeof(agtype_in_state));
 
-    result.res = push_agtype_value(&result.parse_state, WAGT_BEGIN_OBJECT, NULL);
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_BEGIN_OBJECT, NULL);
 
-    result.res = push_agtype_value(&result.parse_state, WAGT_KEY, string_to_agtype_value("id"));
-    result.res = push_agtype_value(&result.parse_state, WAGT_VALUE, integer_to_agtype_value(vertex_id));
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_KEY,
+                                   string_to_agtype_value("__id__"));
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_VALUE,
+                                   integer_to_agtype_value(vertex_id));
 
     for (i = 0; i<fields_len; i++) {
-        result.res = push_agtype_value(&result.parse_state, WAGT_KEY, string_to_agtype_value(header[i]));
-        result.res = push_agtype_value(&result.parse_state, WAGT_VALUE, string_to_agtype_value(fields[i]));
+        result.res = push_agtype_value(&result.parse_state,
+                                       WAGT_KEY,
+                                       string_to_agtype_value(header[i]));
+        result.res = push_agtype_value(&result.parse_state,
+                                       WAGT_VALUE,
+                                       string_to_agtype_value(fields[i]));
     }
 
-    result.res = push_agtype_value(&result.parse_state, WAGT_END_OBJECT, NULL);
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_END_OBJECT, NULL);
 
     return agtype_value_to_agtype(result.res);
 }
 
-agtype* create_agtype_from_list_i(char **header, char **fields, size_t fields_len, size_t start_index) {
+agtype* create_agtype_from_list_i(char **header, char **fields,
+                                  size_t fields_len, size_t start_index) {
 
     agtype_in_state result;
     size_t i;
 
-    if (start_index + 1 == fields_len)
+    if (start_index + 1 == fields_len) {
         return create_empty_agtype();
-
-
+    }
     memset(&result, 0, sizeof(agtype_in_state));
 
-    result.res = push_agtype_value(&result.parse_state, WAGT_BEGIN_OBJECT, NULL);
+    result.res = push_agtype_value(&result.parse_state,
+                                   WAGT_BEGIN_OBJECT, NULL);
 
     for (i = start_index; i<fields_len; i++) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{